code/game/machinery/suit_storage_unit_vr.dm

```diff
@@ -17,3 +17,12 @@
 		SPECIES_ZORREN_FLAT,
 		SPECIES_ZORREN_HIGH
 	)
+
+/obj/machinery/suit_cycler/explorer
+	name = "Explorer suit cycler"
+	model_text = "Exploration"
+	req_access = list(access_pilot)
+	departments = list("Exploration","Pilot")
+
+/obj/machinery/suit_cycler/explorer/initialize()
+	species -= SPECIES_TESHARI
```

Comment:
This will casue errors. You MUST return the return value of ..() in `initialize()`, either via `. = ..()` above your code, or with `return ..()` below. The SSatoms subsystem relies on the return value and will not work correctly if you do not return one of the valid initialize() return values (which a parent proc will return for you if you just return their return values).